test-utils/src/commonMain/kotlin/mock/bluetooth/device/MockDeviceConnectionManager.kt

```diff
@@ -51,7 +51,7 @@ class MockDeviceConnectionManager(
     val readRssiCompleted = AtomicReference(EmptyCompletableDeferred())
     val performActionCompleted = AtomicReference(CompletableDeferred<DeviceAction>())
     val performActionStarted = AtomicReference(CompletableDeferred<DeviceAction>())
-    private val _handledAction = MutableSharedFlow<DeviceAction>(replay = 1, onBufferOverflow = BufferOverflow.DROP_OLDEST)
+    private val _handledAction = MutableSharedFlow<DeviceAction>(replay = 16, onBufferOverflow = BufferOverflow.DROP_OLDEST)
```

Comment:
I guess this is only changes to bluetooth it self, FYI @thoutbeckers

Reply:
ok, yeah I use this in my test too now (I think it will be fine though), thanks for the heads up